src/app/jobs/weatherevents.job.ts

```diff
@@ -0,0 +1,48 @@
+// import { Mode } from '../../common/types';
+import { Job } from '../../common/job';
+import { IContainer } from '../../common/types';
+import { TextChannel } from 'discord.js';
+
+export class WeatherEventsJob extends Job {
+  // Run once an hour
+  public interval: number = 1000 * 60 * 60;
+  // public interval: number = 1000 * 10;
+  public name: string = 'weather_events';
+
+  constructor() {
+    super();
+  }
+
+  public async execute(container: IContainer) {
+    const channel = (container.guildService.getChannel("weather_events")) as TextChannel;
+    const resp = await container.httpService.get("https://api.weather.gov/alerts/active?area=FL");
+    const data: Weather_Event = resp.data;
+
+    if (!data.features || data.features.length == 0) {
+      return;
+    }
+
+    const relevantData: FeaturesEntity[] = data.features.filter(rD => {
+      const highSeverity = rD.properties.severity === 'Severe' 
+                          || rD.properties.severity === 'Extreme';
+
+      return highSeverity && rD.properties.headline.includes("Orlando")
+    });
+
+    const relevantHeadlines: string[] = relevantData.map(rD => rD.properties.headline)
+
+    channel.send(relevantHeadlines.join("\n")).catch(err => console.log(err));
+  }
+
+}
+export interface Weather_Event {
+  features?: (FeaturesEntity)[] | null;
+}
+
+export interface FeaturesEntity {
+  properties: Properties;
+}
+export interface Properties {
+  severity: string;
+  headline: string;
+}
```

Comment:
Is your branch up to date with our develop branch? It should be giving you a linting error for not prefixing interface names with "I"

Reply:
I did git pull inside the repo a couple of days ago before I worked on it. Not sure why it's not working past that I'm not working on the develop branch. I'll change it manually, but can you elaborate on this further?

Reply:
Oh, your branch should be based from `develop` not `master.`. Your PR should also be edited to be merging into `develop`.

Reply:
Oh I see. I changed it to develop.

Reply:
It says it can't find definitions for the linting rules.
![image](https://user-images.githubusercontent.com/49847280/123537879-d2687e00-d6ff-11eb-8b66-7798bcf79cfd.png)